t/305-search.t

```diff
@@ -36,11 +36,6 @@ subtest "search index generation" => {
     }
 }
 
-subtest "Final dot entry" => {
```

Comment:
Related to above. You also eliminate a test. In general, there must be a very good reason to do so. I guess that since URLs are now simply routes in a REST app, you can indeed handle this seamlessly. However, that's not happened yet and in the interim it will simply break several URLs in the static site.

Reply:
Reverted.